01_Exercises/LAB_03.py

```diff
@@ -0,0 +1,171 @@
+import os
```

Comment:
Nota: Se vede ca ai folosit mai multe returnuri la laboratorul asta. Ma andesc ca a fost natural si nu e neaparat ceva ce ti-ai propus. Asta e un avantaj pentru Test Driven Development pentru ca te forteaza sa scrii codul sa fie testabil

Reply:
Asta cu returnurile mi-a intrat in reflex de la codewars. Stii ca ti-am zis ca am stat odata 4 ore si eram sigur ca codul era bun doar ca printam si nu returnam. Atunci am inteles si eu ce face return-ul cu adevarat. 
Mie mi se pare logic acum sa returnezi o valoare fie ea string, bineinteles daca nu avem instructiuni clare sa printam. Desi aici tot am returnat :))